includes/wikia/DefaultSettings.php

```diff
@@ -1002,6 +1002,20 @@
  */
 $wgAdDriverA9OptOutCountries = null;
 
+/**
+ * @name $wgAdDriverA9BidRefreshingCountries
+ * List of countries where A9 bids can refreshed
+ * ONLY UPDATE THROUGH WIKI FACTORY ON COMMUNITY - it's an instant global.
+ */
+$wgAdDriverA9BidRefreshingCountries = null;
```

Comment:
We have to define both variables as instant globals

Reply:
I forgot to remove the dollar sign from search when grepped through the repo 👍